example/yarn.lock

```diff
@@ -4747,6 +4747,11 @@ react-native-haptic-feedback@2.3.1:
   resolved "https://registry.yarnpkg.com/react-native-haptic-feedback/-/react-native-haptic-feedback-2.3.1.tgz#2ef4ac7d4f63ac06bd64b659f509362f127b16e5"
   integrity sha512-dPfjV4iVHfhVyfG+nRd88ygjahbdup7KFZDM5L2aNIAzqbNtKxHZn5O1pHegwSj1t15VJliu0GyTX7XpBDeXUw==
 
+react-native-is-edge-to-edge@^1.0.0:
```

Comment:
Another point - you should also run `yarn` command in `FabricExample` to update `yarn.lock` there as well 👀